### DISCUSSION TITLE
AutoSyncing with GrandPrix Radio

### DISCUSSION THREAD

**Discussion by RikoSakura** [f1multiviewer/issue-tracker#282] [Feature requests] [answered]
Autosyncing would be a nice feature to have. just so i can open grand prix radio and it's timed live already and i don't have to do anything. ofcourse i don't know if it's possible but it would be a really nice feature to have.

**Comment by spambas** [comment, marked as answer]
Issue is the grandprix radio stream does not have any markers the video's could be synced to. 
as long as they wont change their way of streaming this will not be possible.

**Comment by JJWatMyself** [comment]
With the updated official F1 Live Timing web site you can do a manual sync with pause play.  It's the same radio feed from BBC Radio 5 Live.  Is that the Grand Prix Radio you are talking about?

**Reply by alexedwards565** [reply]
I'd love to be able to sync the 5 live coverage with the video
